.dockerignore

```diff
@@ -24,10 +24,21 @@ docs
 .cache
 .circleci
 .git
+.github
 .vscode
+.ssh
+crowdin
+lib/gitlint
 
 # Assets
 data
+src/backend/locale
```

Comment:
This directory is not re-generated by the Dockerfile and should not be excluded ?

Reply:
Good catch ! Thanks